app/models/account/provider_domains.rb

```diff
@@ -49,13 +51,15 @@ def find_by_domain(domain)
       find_by(domain: domain)
     end
 
+    # rubocop:disable Rails/DynamicFindBy
```

Comment:
I think that it can be done only for the specific line instead of the whole method just as u did in https://github.com/3scale/porta/pull/1097/files#diff-7be675ec904f1d3a73803f8bd354afb1R68